pkgs/shells/marcel/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, dill
+, psutil
+, makeWrapper
+, python3
+, bash
+}:
+
+buildPythonPackage rec {
```

Comment:
It might be more appropriate to use [buildPythonApplication](https://ryantm.github.io/nixpkgs/languages-frameworks/python/#buildpythonapplication-function).
If you do this you'll have to do `callPackage` instead of `python3Packages.callPackage` in `all-packages.nix`.
Look at the linked page for more info.

Reply:
thank you! now change to use buildPythonApplication.